### PR TITLE
[imp]set GO111MODULE=on in the Makefile by default

### PR DESCRIPTION
**Pull request by scaat**
### Issues associated with this PR

Your PR should present related issues you want to solve.
#1720
### Solutions
在Makefile中全部启用`GO111MODULE=on`，同时选择使用`golang`镜像，跳过构建中间层的`godep-builder`镜像，`godep-builder`的存在，只是为了在默认`golang`镜像中将`GO111MODULE`设置为`off`，全部启用`GO111MODULE=on`后，就不需要这层中间镜像。

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases. And you need to show the UT result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result.

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -35,46 +35,39 @@ ifneq ($(TAGS),)
 TAGS_OPT 		= -tags ${TAGS}
 endif
 
-
 ut-local:
-	GO111MODULE=off go test -gcflags=-l -v `go list ./pkg/... | grep -v pkg/mtls/crypto/tls | grep -v pkg/networkextention`
+	GO111MODULE=on go test -gcflags=-l -v `go list ./pkg/... | grep -v pkg/mtls/crypto/tls | grep -v pkg/networkextention`
```
Comment (doujiang24) [inline Makefile:39]: does go test will use the modules in vendor?

Comment (scaat) [reply Makefile:39]: yes

**Comment by nejisama** [comment]
按照ISSUE里讨论的，默认开启GO111MODULE的话，需要新增-mod=vendor 才对吧

**Comment by scaat** [comment]
是的，是我疏忽了，已经重新更新了。 @nejisama

**Review by nejisama — APPROVED** [review]

**Review by doujiang24 — APPROVED** [review]

**Pull request merged**